src/unittest/test_modmetadatadatabase.cpp

```diff
@@ -104,6 +114,49 @@ class SQLite3Provider : public ModMetadataDatabaseProvider
 	std::string dir;
 	ModMetadataDatabase *mod_meta_db = nullptr;
 };
+
+#if USE_POSTGRESQL
+void clearPostgreSQLDatabase(const std::string &connect_string)
+{
+	std::unique_ptr<PGconn, decltype(&PQfinish)> conn(
+			PQconnectdb(connect_string.c_str()), PQfinish);
+	if (PQstatus(conn.get()) == CONNECTION_OK) {
+		std::unique_ptr<PGresult, decltype(&PQclear)> res(
+				PQexec(conn.get(), "DROP TABLE IF EXISTS mod_storage;"), PQclear);
+		if (PQresultStatus(res.get()) == PGRES_COMMAND_OK)
+			return;
+	}
+	throw DatabaseException(
+			std::string("PostgreSQL database error: ") + PQerrorMessage(conn.get()));
+}
```

Comment:
for simplicity reasons don't we want to make this a `listMods` + `removeModEntries`?